Cargo.toml

```diff
@@ -1,16 +1,16 @@
 [package]
 
 name = "webdriver"
-version = "0.19.0"
+version = "0.19.1"
 authors = ["Mozilla Tools and Automation <auto-tools@mozilla.com>"]
 description = "Library implementing the wire protocol for the W3C WebDriver specification."
 repository = "https://github.com/jgraham/webdriver-rust"
 license = "MPL-2.0"
 
 [dependencies]
 backtrace = "0.3"
-cookie = {version = "0.2", default-features = false}
-hyper = {version = "0.9", default-features = false}
+cookie = {version = "0.6", default-features = false}
+hyper = "0.10"
```

Comment:
This presumably enables SSL to be compiled into hyper, which will fail when we cross compile [geckodriver](https://github.com/mozilla/geckodriver) to Windows x86 and x86_64.